openml_OS/views/pages/api_new/v1/xml/study-attach-detach.tpl.php

```diff
@@ -0,0 +1,6 @@
+<oml:study_<?php echo $function_type; ?> xmlns:oml="http://openml.org/openml">
+  <oml:id><?php echo $id; ?></oml:id>
+  <oml:main_knowledge_type><?php echo $main_knowledge_type; ?></oml:main_knowledge_type>
```

Comment:
You called this an entity type earlier, and now a knowledge type?
Why 'main'? A study has 1 type, right?

Reply:
done